### ISSUE TITLE
Training my own dataset (radar point dataset) failed

### ISSUE THREAD

**Issue by ghost** [dgriffiths3/pointnet2-tensorflow2#8]
Hi,

I'd like first of all to thank you for publishing your code, it's very helpful. I am currently trying to train my own dataset using PointNet++ and semantic segmentation model. I tried to convert my data(data that I split myself into training and validation )to tf record. 
I have labels for training and validation and also points for training and validation, both into arrays.
Labels is an np array containing the indexes of labels of type int64 and Points is an np.array containing points corresponding to each label of type float32

This way:

```
def tables_to_TF( Label,Points,tf_filename):
    
    filepath = os.path.join(tf_filename)
    print('Writing', filepath)
    writer = tf.io.TFRecordWriter(tf_filename)
    example = tf.train.Example(
            features=tf.train.Features(feature={
            'Points': _float_feature(Points),
            'Labels': _int64_feature(Label),
            
                }) 
                    )
    
    writer.write(example.SerializeToString())

```

But loading the data does not work, I get can't sparse serialized Example.
Now when trying to train the dataset using your code I get this:

![image](https://user-images.githubusercontent.com/71985056/94372984-7b972f00-0102-11eb-8503-4c2b9bd86e9d.png)

I don't understand why it does not work and why the conversion doesn't work. I'm stuck and I have tried everything but could not resolve the problem.

It will be very kind of you if you could help me. I'm counting on your help.

Thank you in advance.

Kind regards!

**Comment by pratibhashinde** [comment]
Hi there,
Did you solve this issue? I am also facing the same issue when loading my dataset. I have .h5 files for the data and I converted them into .tfrecords. Still the same error is coming. Please help me. 

Thanks.
